### PR TITLE
Download and import curated packages package bundle by default

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*
- Fetch curated packages bundle by default since it is needed by package controller to function properly.
*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by chrisdoherty4 — APPROVED** [review]
Just for clarity, is this in-line with the move to install the components necessary for package management by default?

File: release/api/v1alpha1/artifacts.go [hunk 1]
```diff
@@ -222,6 +222,7 @@ func (vb *VersionsBundle) Charts() map[string]*Image {
 	return map[string]*Image{
 		"cilium":                &vb.Cilium.HelmChart,
 		"eks-anywhere-packages": &vb.PackageController.HelmChart,
+		"ecr-token-refresher":   &vb.PackageController.TokenRefresher,
```
Comment (TerryHowe) [inline release/api/v1alpha1/artifacts.go:225]: I'm confused, this isn't a chart

Comment (a-cool-train) [reply release/api/v1alpha1/artifacts.go]: Ok removed it.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**